### PR TITLE
Fix multiple memory leak issues

### PR DESCRIPTION
**Pull request by ivq**

### CODE REVIEW

File: options.c [hunk 1]
```diff
@@ -296,6 +296,7 @@ options_remove(struct options_entry *o)
 	else
 		options_value_free(o, &o->value);
 	RB_REMOVE(options_tree, &oo->tree, o);
+	free((void *)o->name);
```
Comment (krader1961) [inline options.c]: You do not need, and should not include, the cast. The few places in the code which do this cast should remove the cast, IMHO. This cast hasn't been needed for more than two decades.

File: tmux.c [hunk 2]
```diff
@@ -127,6 +127,7 @@ make_label(const char *label, char **cause)
 		free(base);
 		goto fail;
 	}
+    free(base);
```
Comment (krader1961) [inline tmux.c]: This is a partial fix. Notice the other places that `goto fail;`. This is a situation where it is better to initialize the var to `NULL` where it is defined and then do `if (base) free(base);` in the error path.

File: options.c (hunk 1 above)
Comment (nicm) [reply options.c]: These are const casts and are necessary.

File: tmux.c (hunk 2 above)
Comment (nicm) [reply tmux.c]: Yes that would work, although free(NULL) is always OK so no need for an if guard.

Comment (ivq) [reply tmux.c]: Note the spaces here. Tab should be used.

**Comment by nicm** [comment]
Applied now, thanks!

**Pull request closed without merging**